packages/wrangler/src/pages.tsx

```diff
@@ -795,22 +798,19 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
             functionsDirectory,
             sourcemap: true,
             watch: true,
-            onEnd: () => {
-              scriptReady = true;
-            },
+            onEnd: () => scriptReadyResolve(),
           });
 
           watch([functionsDirectory], {
             persistent: true,
+            ignoreInitial: true,
```

Comment:
Nit: rather than including this line, we could just remove L796–802, if I'm understanding what this does correctly.

Reply:
Ahh, gotcha. Sure. Just leave as-is :)